### ISSUE TITLE
Noise Adding Iteration and Denoising Iteration

### ISSUE THREAD

**Issue by DWF09** [tencent-ailab/MDM#10]
Hi, I don't see noise addition and denoising for t time steps in your training code, I only see noise addition for one time step, no iterative noise addition, maybe I didn't read your code completely, can you explain my confusion?

**Comment by trice-chance** [comment]
Actually, just the loss convergence, don't need diffusion number steps, just repeat until convergence
